CONTRIBUTING.md

```diff
@@ -39,46 +36,95 @@ Check out the requisites before contributing to Horusec:
 
  This is a security layer for the project and for the developers. It is mandatory.
  
- There are two ways to use DCO, see them below: 
+ Follow one of these two methods to add DCO to your commits:
  
 **1. Command line**
  Follow the steps: 
- **Step 1:** Check out your local git:
+ **Step 1:** Configure your local git environment adding the same name and e-mail configured at your GitHub account. It helps to sign commits manually during reviews and suggestions.
 
  ```
 git config --global user.name “Name”
 git config --global user.email “email@domain.com.br”
 ```
-**Step 2:** When you commit, add the sigoff via `-s` flag:
+**Step 2:** Add the Signed-off-by line with the `'-s'` flag in the git commit command:
 
 ```
 $ git commit -s -m "This is my commit message"
 ```
 **2. GitHub website**
 
-**Step 1:** When the commit changes box opens, add 
+You can also manually sign your commits during GitHub reviews and suggestions, follow the steps below: 
+
+**Step 1:** When the commit changes box opens, manually type or paste your signature in the comment box, see the example:
+
 ```
-$ git commit -m “My signed commit” Signed-off-by: username <email address>
+Signed-off-by: Name < e-mail address >
 ```
-Note: For this option, your e-mail must be the same in registered in GitHub. 
+
+For this method, your name and e-mail must be the same registered on your GitHub account.
 
 ### **Code Review**
 - All your submissions needs a review.
 
 ### **Pull Requests**
-When opening a PR: 
-- You need to add a title describing the issue. 
-- Fill in the template, describe why you are opening the PR.
+When you open a Pull Request, follow the requirements below:
+
+1. Add a title with the following pattern: 
+
+#### **[PKG][TYPE]: Description**
+
+#### **PKG:** Name of the package or main service you want to change.
+
+#### **TYPE**: Add what your Pull Request (PR) refers to:
+- **FEATURE:** PR refers to a new activity.
+- **BUGFIX:** PR refers to corrections for the next release.
+- **HOTFIX:** PR refers to corrections where you will need a cherry-pick and the update of the minor version. 
+- **CHORE:** PR refers to changes for the next release, but it was only maintenance without an activity impact.  
+
+**Example:** **[start][bugfix]: Fix bug when Horusec haven't read the new flag of authorization**
```

Comment:
The pattern is without `[]`, for example: `start:bugfix: fix bug when Horusec haven't read the new flag of authorization`

Take a look at current open PRs https://github.com/ZupIT/horusec/pulls